neurodsp/sim/cycles.py

```diff
@@ -0,0 +1,275 @@
+"""Simulating individual cycles, of different types."""
+
+import numpy as np
+from scipy.signal import gaussian, sawtooth
+
+from neurodsp.utils.checks import check_param
+from neurodsp.sim.info import get_sim_func
+from neurodsp.sim.transients import sim_synaptic_kernel
+
+###################################################################################################
+###################################################################################################
+
+def sim_cycle(n_seconds, fs, cycle_type, phase=0, **cycle_params):
```

Comment:
`phase` isn't used in this func.

Reply:
ahh, good catch! At one point I applied phase rotation here, but I think it makes more sense in sim_periodic. I've dropped this argument from here.